### ISSUE TITLE
Non-existent sequences sometimes return inappropriate message (perhaps on second try to access?)

### ISSUE THREAD

**Issue by katestange** [numberscope/backscope#104]
Create a test for the situation in which a request is made for terms (or other endpoints?) for an A-number that does not exist in the OEIS.

**Comment by gwhitney** [comment]
The logging test that attempts to get A000000 seems to qualify for this; @katestange should this issue be closed?

**Comment by katestange** [comment]
This does appear to be addressed, in that there is a test that calls for such a sequence; but it tests that a log file is created, right?  Right now the API call `http://127.0.0.1:5000/api/get_oeis_values/A000000/50` returns `Error: B-file for ID 'A000000' not found in OEIS.`.  I think this is good behaviour.  But are we testing this behaviour, i.e. that we are returning this correct error message (the one above) to the API call?  That would be a good thing to test, right?

**Comment by gwhitney** [comment]
Yes, I agree. That would be a good thing to test, especially since now when you access `https://numberscope.colorado.edu/api/get_oeis_values/A000000/100` the result is `Error: Value fetching for {oeis_id} in progress.` which is _not_ a good response. So changing this to a bug, and presumably testing will be a part of fixing the bug. Priority of this with respect to students starting?

**Comment by katestange** [comment]
Oh interesting, so the first time you get a good response and the second time you get a bad response?

**Comment by katestange** [comment]
As for priority.  It's probably a decent priority?  It's not something they'll run into a lot, but it's not good.

**Comment by gwhitney** [comment]
OK, I will put it near the bottom of the project list.

**Comment by Vectornaut** [comment]
> This does appear to be addressed, in that there is a test that calls for such a sequence; but it tests that a log file is created, right?

The test checks both the expected response ([line 20](https://github.com/numberscope/backscope/blob/fb06fd1ad46a81525f9af9ea7610982a0b8ba9ef/flaskr/nscope/test/test_lookup_errors.py#L20)) and the expected log entry (lines [21–28](https://github.com/numberscope/backscope/blob/fb06fd1ad46a81525f9af9ea7610982a0b8ba9ef/flaskr/nscope/test/test_lookup_errors.py#L21)).

> Right now the API call http://127.0.0.1:5000/api/get_oeis_values/A000000/50 returns `Error: B-file for ID 'A000000' not found in OEIS.`.

This is a manifestation of #69. It's described in more detail in the ['Manual tests"](https://github.com/numberscope/backscope/blob/main/doc/logging.md#manual-tests) section of the logging docs. (Asking to put that stuff in the docs was a good idea!)

**Comment by gwhitney** [comment]
Great, so when you squash #69 this should hopefully come along for the ride!

**Comment by Vectornaut** [comment]
That's the plan!
